exe-unit/src/counters/os/unix.rs

```diff
@@ -101,3 +104,9 @@ impl Metrics {
             })
     }
 }
+
+impl From<SystemError> for MetricError {
+    fn from(err: SystemError) -> Self {
+        Self::Other(err.to_string())
```

Comment:
1. `unix.rs` is still inside `exe-unit`. I will move it to `ya-counters` later - I wanted to have _custom_ implementation of `Metrics` outside of `ya-counters` before implementing `Metrics` for `ya-runtime-ai`.
2. I did not wanted to have generics in `MetricError` so I just translate it to `Other`.